spec/util/string_to_boolean_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+require 'cfn-nag/util/string_to_boolean'
+
+describe 'to_boolean', :rule do
+
+  context 'Strings map to boolean' do
+    it 'given any true string, to_boolean returns true boolean' do
+      %w[true TRUE tRUE True tRuE TrUe]
```

Comment:
is it confirmed that cfn is cool with these other cases?  (i presume so, just checking)

Reply:
Update: It's not! Good catch. I was testing the function's tactic rather than the overarching strategy. Refactoring.